packages/tradershub/src/features/cfd/modals/MT5PasswordModal/MT5Success.tsx

```diff
@@ -17,6 +17,7 @@ const MT5SuccessModal = () => {
     const { isEU } = useRegulationFlags();
     const { data: mt5Accounts } = useMT5AccountsList();
     const { data: activeTrading } = useActiveTradingAccount();
+
```

Comment:
uh noice

Reply:
If you want to keep it , it's fine no judgement here but if you remove the space you have one less file inside your changes.

Reply:
astaga